public/wp-config.php

```diff
@@ -50,6 +50,8 @@
 $imgproxy = $params['imgproxy'];
 define('IMGPROXY_KEY', $imgproxy['key']);
 define('IMGPROXY_SALT', $imgproxy['salt']);
+define('IMGPROXY_PROJECT_ID', $imgproxy['projectId'] ?? 'CHANGE_THIS');
+define('IMGPROXY_BASE_URL', rtrim(($imgproxy['baseUrl'] ?? 'https://snappycdn.net'), '/') . '/' . IMGPROXY_PROJECT_ID);
```

Comment:
vzhledem k tomu že to není konstanta tak bych z toho udělal funkci nebo proměnnou